layers/meta-opentrons/recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -50,12 +50,14 @@ MAX_SYSTEMFS_SIZE = "1536"
 # create the opentrons ot3 manifest (VERSION.json) file
 python do_create_opentrons_manifest() {
     bb.note("Create the manifest json for for ot3-system.zip")
+    import time
     import json
     import os
 
     opentrons_manifest = {
         'robot_type': d.getVar('ROBOT_TYPE'),
         'build_type': os.getenv('OT_BUILD_TYPE', 'unknown/dev'),
+        'build_timestamp': time.time(),
```

Comment:
let's not do this - it guarantees that subsequent builds that otherwise have the same content will not be content-equal, ruining any prospect for [build hermeticity](https://bazel.build/basics/hermeticity) (note that we almost certainly don't currently have hermetic builds, but it's a good goal to have). If we can identify the versions of everything else, we fundamentally don't really care about when the build occurred.